.github/actions/update-dockerhub-readme/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Update Docker Hub README'
+description: 'Composite action to update Docker Hub README for OSS and EE images'
+inputs:
+  should_build_readme_oss:
+    description: 'Flag to determine if OSS README should be updated'
+    required: true
+  should_build_readme_ee:
+    description: 'Flag to determine if EE README should be updated'
+    required: true
+  docker_username:
+    description: 'Docker username'
+    required: true
+  docker_password:
+    description: 'Docker password'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout Code
```

Comment:
Why does this action need to `checkout`? if it's called from another action, mustn't it already be checked out?

Reply:
Good finding, fixed: https://github.com/hazelcast/hazelcast-docker/pull/793/commits/92183e18e9f56fc49d76829f1dcb0106de76b7f5. It was a leftover after my experiments with reusable workflows